Documentation/AdvancedUsage.md

```diff
@@ -653,7 +665,12 @@ let session = Session(eventMonitors: [logger])
 ```
 
 ## Making Requests
-As a framework, Alamofire has two main goals: to enable the easy implementation of network requests for prototypes and tools, and to serve as the generic foundation of app networking. It accomplishes these goals through the use of powerful abstractions, providing useful defaults, and included implementations of common tasks. However, once use of Alamofire  has gone beyond a few requests, it’s necessary to move beyond the high level, default implementations into behavior customized for particular applications. Alamofire provides the `URLConvertible` and `URLRequestConvertible` protocols to help with this customization.
+As a framework, Alamofire has two main goals: 
+
+1. To enable the easy implementation of network requests for prototypes and tools
+2. To serve as the generic foundation of app networking
```

Comment:
I thought we could put a finer point on it by actually listing them out.

Reply:
Looks good. Do these need periods?